code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -811,7 +834,8 @@ GLOBAL_LIST_EMPTY(PDAs)
 		if(old_id)
 			user.put_in_hands(old_id)
 		update_icon()
-		playsound(src, 'sound/machines/button.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/button.ogg', 15, 1)

```

Comment:
Sound is already quiet, volume change can be reverted. Also, this is a mechanical sound, making this mutable via a software toggle makes no logical sense